sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1403,27 +1417,57 @@ def expand(self, input):
         retry_strategy=self.retry_strategy,
         test_client=self.test_client,
         additional_bq_parameters=self.additional_bq_parameters,
-        ignore_insert_ids=self.ignore_insert_ids)
+        ignore_insert_ids=self.ignore_insert_ids,
+        with_batched_input=(
+            not self.ignore_insert_ids and self.with_auto_sharding))
```

Comment:
how come `ignore_insert_ids` affects this parameter?

Reply:
Good catch!! Removed.